x-pack/legacy/plugins/spaces/server/lib/space_schema.ts

```diff
@@ -19,4 +19,7 @@ export const spaceSchema = Joi.object({
     .items(Joi.string())
     .default([]),
   _reserved: Joi.boolean(),
+  imageUrl: Joi.string()
+    .allow('')
+    .regex(/^data:image.*$/, `Image URL should start with 'data:image'`),
```

Comment:
Let's make sure to add the appropriate [schema tests](https://github.com/elastic/kibana/blob/b8303e4cb153b682bca4d16819f280c6284d2a33/x-pack/legacy/plugins/spaces/server/lib/space_schema.test.ts) for this change too

Reply:
Done.